apps/site/next.fonts.ts

```diff
@@ -1,21 +1,19 @@
-import { Open_Sans, IBM_Plex_Mono } from 'next/font/google';
+import localFont from 'next/font/local';
 
 // This configures the Next.js Font for Open Sans
 // We then export a variable and class name to be used
 // within Tailwind (tailwind.config.ts) and Storybook (preview.js)
```

Comment:
Can you link the file source?

Reply:
Incorrect; I got'em from some random GitHub repositories xD

Reply:
- [Open Sans](https://github.com/edx/ux-pattern-library/blob/master/pattern-library/fonts/OpenSans/OpenSans-Regular-webfont.woff2)
- [IBM Plex Mono](https://github.com/qiskit-community/qiskit.camp/blob/master/fonts/IBM-Plex-Mono/fonts/split/woff2/IBMPlexMono-Regular-Latin2.woff2)